conf/eslint.json

```diff
@@ -80,6 +80,7 @@
         "no-process-env": "off",
         "no-process-exit": "off",
         "no-proto": "off",
+        "no-prototype-built-ins": "off",
```

Comment:
I would make it `no-prototype-builtins`.

Prior art:

- http://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html
- https://docs.python.org/3/library/builtins.html